src/utils/api.ts

```diff
@@ -238,10 +238,6 @@ function invalidCsrfResponse(res) {
  */
 export async function validateAddress(req, res, appObj = app) {
   const tokenObject = appObj["tokenObject"];
-  const csrfTokenValid = validateCsrfToken(req);
```

Comment:
Remove this same code from `validateUsername`.

Reply:
That's true, but it's also true for the address endpoint. That one is now exposed and both are authenticated requests to platform api. Someone could make multiple hits to the internal address endpoint and (for whatever reason) validate addresses:

<img width="811" alt="Screenshot 2025-04-07 at 4 40 46 PM" src="https://github.com/user-attachments/assets/f36d8ce1-ea42-47c0-b175-a7092348a404" />

So this would be true for the prod url `https://www.nypl.org/library-card/api/address`

Reply:
There's concern for both endpoints, but we do pay for an address validation service. It would be a problem if that gets overloaded with requests.

Reply:
ok! I replaced it on the address validation.